spring-cloud-data-module-deployers/spring-cloud-data-module-deployer-spi/src/main/java/org/springframework/cloud/data/module/ModuleStatus.java

```diff
@@ -60,6 +60,11 @@
 		deployed,
 
 		/**
+		 * The module exited normally
```

Comment:
No.  Tasks can't be scaled.  That's a limitation of Diego, but one I think we should be ok with running with on our side as well.